src/components/App/App.js

```diff
@@ -18,22 +19,30 @@ class App extends Component {
   render() {
     return (
       <main>
-      <Route
-        exact
-        path="/"
-        render={() => (
-          <MovieContainer
-            movies={this.props.movies}
-          />
-        )}
-      />
+        <Nav
+          userInfo={this.props.userInfo}
+        />
+        <Route
+          exact
+          path="/"
```

Comment:
Smart choice to make this exact. We don't want to over-use exact, but it's helpful here because we don't want any url with / to trigger this route.